dnf-behave-tests/fixtures/specs/needs-restarting-updates/abcde-2.9.3-1.fc29.spec

```diff
@@ -0,0 +1,22 @@
+Name:           abcde
+Version:        2.9.3
+Release:        1%{?dist}
+Summary:        A Better CD Encoder
+
+# cddb-tool is Public Domain, otherwise GPLv2+
+License:        GPLv2+ and Public Domain
+URL:            https://abcde.einval.com/
+
+BuildArch:      noarch
+Requires:       wget
+Recommends:     flac
+Suggests:       lame
```

Comment:
There are attributes on the packages like the above which are irrelevant to the test and therefore superfluous (also the lengthy `%description`s). Again, not hurting anything and they are not too long, so feel free to keep them but in general it's good practice to keep the test data minimal to the test case.